src/com/content-games/grid-selector.js

```diff
@@ -0,0 +1,81 @@
+import { h, Component } 				from 'preact/preact';
+
+import Dropdown							from 'com/input-dropdown/dropdown';
+import ButtonBase						from '../button-base/base';
+import SVGIcon 							from 'com/svg-icon/icon';
+
+export default class GridSelector extends Component {
+    constructor( props ) {
+		super(props);
+        this.state = {
+            expanded: false,
+            selected: null,
+        };
+
+        this.onSelectLayout = this.onSelectLayout.bind(this);
+        this.onToggleDropDown = this.onToggleDropDown.bind(this);
+    }
+
+    componentDidMount() {
+       if (!this.state.selected) {
+          this.handleNewSelection(this.props.defaultLayout);
+       }
+    }
+
+    handleNewSelection(selected) {
+        const onChangeLayout = this.props.onChangeLayout;
+        this.setState({selected: selected});
+        if (onChangeLayout) {
+            onChangeLayout(selected);
+        }
+    }
+
+    onToggleDropDown() {
+        console.log('toggle dropdown', this.state);
+        this.setState({expanded: !this.state.expanded});
+    }
```

Comment:
Please wrap member names in string quotes. i.e. `{'expanded': ...}`. For consistency with JSON.